tools/doxygen/headerFile

```diff
@@ -0,0 +1,77 @@
+<!-- HTML header for doxygen 1.13.2-->
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="$langISO">
+<head>
+<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
+<meta http-equiv="X-UA-Compatible" content="IE=11"/>
+<meta name="generator" content="Doxygen $doxygenversion"/>
+<meta name="viewport" content="width=device-width, initial-scale=1"/>
+<!--BEGIN PROJECT_NAME--><title>$projectname: $title</title><!--END PROJECT_NAME-->
+<!--BEGIN !PROJECT_NAME--><title>$title</title><!--END !PROJECT_NAME-->
+<link rel="icon" href="/icon_black.png" media="(prefers-color-scheme: light)">
+<link rel="icon" href="/icon_white.png" media="(prefers-color-scheme: dark)">
```

Comment:
Observation: This only works on icu4x.unicode.org, not in other places where this doxyhtml could be hosted. Is there a nice way to fix?

I think I'm not opposed to shipping the icon files again in this subdirectory